### PR TITLE
Cl-8-YS read only feature

### PR DESCRIPTION
**Pull request by Yash-Shukla-9**
1. Fix: Fixed the navbar which was not showing earlier.
2. Style: Added some padding to the code editor for better viewing.

### CODE REVIEW

**Comment by Yash-Shukla-9** [comment]
@xDAnkit Sir Please check the PR.

File: client/src/HomePage.jsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ export default function HomePage() {
         alignItems: "center",
       }}
     >
-      <Link to={`/code/${id}`} style={{ textDecoration: "none", color: "black" }}>
+      <Link to={`/${id}`} style={{ textDecoration: "none", color: "black" }}>
```
Comment (xDAnkit) [inline client/src/HomePage.jsx]: @Yash-Shukla-9 - Please remove inline css

**Review by xDAnkit — CHANGES_REQUESTED** [review]
@Yash-Shukla-9 - Please review comments

**Review by Yash-Shukla-9 — COMMENTED** [review]
All set to go

File: client/src/HomePage.jsx [hunk 2]
```diff
@@ -15,7 +15,7 @@ export default function HomePage() {
         alignItems: "center",
       }}
     >
-      <Link to={`/code/${id}`} style={{ textDecoration: "none", color: "black" }}>
+      <Link to={`/${id}`} className="Button-link">
```
Comment (xDAnkit) [inline client/src/HomePage.jsx]: @Yash-Shukla-9 - Class name should be in correct format `btn-link`

**Review by xDAnkit — CHANGES_REQUESTED** [review]
@Yash-Shukla-9 - Check comments

**Review by Yash-Shukla-9 — COMMENTED** [review]
Fixed the style

**Review by xDAnkit — APPROVED** [review]
Looks good to me

**Pull request merged**